rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
```

Comment:
- [ ] Here, you are not setting the associations properly. Right now, you need to set the "belongs-to" association between book and rentals, and the same association between person and rentals. So please make sure to add it.